test/jdk/java/security/cert/PolicyNode/GetPolicyQualifiers.java

```diff
@@ -52,6 +53,8 @@ public static void main(String[] args) throws Exception {
         PKIXParameters params = new PKIXParameters(trustAnchors);
         params.setPolicyQualifiersRejected(false);
         params.setRevocationEnabled(false);
+        // Certificates expired on Oct 6th, 2020
+        params.setDate(new Date("July 01, 2020"));
```

Comment:
The Date() constructor is deprecated, would you like to use the replacement method DateFormat.parse()?

Reply:
I have fixed it, thanks!